### DISCUSSION TITLE
Can't install solidus_starter_fontend

### DISCUSSION THREAD

**Discussion by rega1237** [solidusio/solidus#4687] [Troubleshooting]
Hi everyone! I have a problem when I run `rails g solidus:install` everithing is fine but when I select the solid_starter_frontend as template I get this error:

![solidus error](https://user-images.githubusercontent.com/58194121/197399469-93b45446-7f19-47ff-9317-58a4f757ebd5.png)

ruby version: 3.1.2p20
Rails version 7.0.4

Thanks in advance

**Comment by waiting-for-dev** [comment]
Thanks for your report, @rega1237. We're looking into it and will let you know what we find out.

**Comment by gsmendoza** [comment]
@rega1237 Are you still having the issue? I was able to install Solidus with SolidusStarterFrontend on Ruby 3.1.2. Please see https://github.com/gsmendoza/solidus_store/tree/rails-7-0-4--solidus-3-2-2--ruby-3-1-2.

**Comment by elia** [comment]
I think this might have something to do with the shell available in windows, can verify lack of support for inline environment variables with this simple experiment?

- open `irb`
- run `system 'FOO=123 echo $FOO'`

does it print `123` it complains about a missing `FOO=123` file like in the original error?

**Reply by elia** [reply]
@rega1237 I just opened a PR with a potential fix, can you try that? 

In order to do that you need to add solidus to the gemfile pointing to the PR branch:

```rb
gem 'solidus', git: 'https://github.com/nebulab/solidus.git', branch: 'elia/3-2-inline-env-fix'
```

**Reply by Fahmiin-Abdullah** [reply]
@elia I got this

![image](https://user-images.githubusercontent.com/64894579/198844618-024d43e9-86ce-4a43-bdad-bef871079ff3.png)

**Comment by Fahmiin-Abdullah** [comment]
@elia I'm getting the same problem when trying to run `rails g solidus:install` and choosing the new frontend. This is my first time trying out solidus

I have to interrupt and exit otherwise it'll get stuck here forever

![image](https://user-images.githubusercontent.com/64894579/198844385-8b540af7-ee32-408e-89ca-c79db6e48bee.png)

rails 6.1.7
ruby 2.6.5

any fix implemented yet? Thanks!
